core/domain/entities/shortcodes/EspressoEventAttendees.php

```diff
@@ -347,9 +352,15 @@ private function getTicket(array $attributes)
     private function setAdditionalQueryParams(array $attributes)
     {
         $reg_status_array = EEM_Registration::reg_status_array();
-        if ($attributes['status'] !== 'all' && isset($reg_status_array[ $attributes['status'] ])) {
+        if ($attributes['status'] !== 'all'
+            && $attributes['status'] !== 'ANY'
```

Comment:
Although this still works, I'm not sure its needed now because status just won't be set right?